### PR TITLE
Update saucelab endpoint in fastline

### PR DESCRIPTION
**Pull request by Serhy**
Update saucelab endpoint in fastline

### CODE REVIEW

**Review by jakubgs — APPROVED** [review]

File: test/appium/tests/base_test_case.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def sauce_access_key(self):
 
     @property
     def executor_sauce_lab(self):
-        return 'https://%s:%s@ondemand.eu-central-1.saucelabs.com:443/wd/hub' % (self.sauce_username, self.sauce_access_key)
+        return 'http://%s:%s@ondemand.eu-central-1.saucelabs.com:80/wd/hub' % (self.sauce_username, self.sauce_access_key)
```
Comment (jakubgs) [inline test/appium/tests/base_test_case.py]: Why tho? HTTP is insecure. Is it failing with HTTPS?

Comment (Serhy) [reply test/appium/tests/base_test_case.py]: Just tried if 433 was the reason of why many tests are failing after that migration (we had it previously on 80th). For example https://ci.status.im/job/end-to-end-tests/job/status-app-prs/1575/console ``` HTTPSConnectionPool(host='ondemand.eu-central-1.saucelabs.com', port=443): Max retries exceeded ```

Comment (jakubgs) [reply test/appium/tests/base_test_case.py]: But the issue appears to not be `Max retries exceeded` but `too many redirects`:
```
urllib3.exceptions.MaxRetryError:
HTTPSConnectionPool(host='ondemand.eu-central-1.saucelabs.com', port=443):
Max retries exceeded with url: https://****:****@ondemand.eu-central-1.saucelabs.com:443/wd/hub/session/2a95c0353773469e9b58b3917d0033d6?ref=ed95194737c64f728cfb5ddd9b23dc00 
(Caused by ResponseError('too many redirects'))
```
Have you tried not specifying the port?

Comment (jakubgs) [reply test/appium/tests/base_test_case.py]: A regular `GET` request works just fine:
```
 > curl -s 'https://ondemand.eu-central-1.saucelabs.com/' 
{"ready":true,"message":"Sauce Labs automated testing is fully operationa
```

Comment (Serhy) [reply test/appium/tests/base_test_case.py]: Yes but it didn't help without port - removed in https://github.com/status-im/status-react/pull/11628/commits/c9b40ab32da9814fa167a7b1051240217de73a8a (but used http there though). Some tests are still failing https://ci.status.im/job/end-to-end-tests/job/status-app-prs-rerun/481/console with this 'too many redirects'.

Comment (Serhy) [reply test/appium/tests/base_test_case.py]: They are not all ERRORing with it, - but those which are failed screw all further threads of tests...

Comment (jakubgs) [reply test/appium/tests/base_test_case.py]: Makes sense. I guess we're fine with HTTP for now, but could we open some kind of Support ticket with SauceLabs to ask about this issue?

Comment (Serhy) [reply test/appium/tests/base_test_case.py]: Already reached their support

Comment (jakubgs) [reply test/appium/tests/base_test_case.py]: Sweet.

**Comment by flexsurfer** [comment]
@Serhy what's this PR about?

**Comment by Serhy** [comment]
It's about change of datacenter endpoint in e2e tests (there are two datacenter we may use in e2e: one based in US and another one in EU). Thing is that when we specified appropriate endpoint (in URL we use to connect to VM cloud service) many tests start failing because they exceed concurrency limit we have in SauceLabs). 
We'll need to switch to to datacenter anyway (now we have a common url which works) because they (Saucelabs) may update endpoint any time. 
It was not urgent but I'll take a look at it later again. Closing for now this.

**Pull request closed without merging**